code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm

```diff
@@ -18,31 +18,31 @@
 	reagents.maximum_volume = 40
 
 
-/*/mob/living/carbon/superior_animal/roach/benzin/Life()
+/mob/living/carbon/superior_animal/roach/benzin/Life()
 	if(reagents.total_volume < reagents.maximum_volume)
 		reagents.add_reagent("fuel", 1)
-	..()*/
+	..()
```

Comment:
This needs to be re-commented out. It'd be appreciated if an `OCCULUS EDIT` comment could be tacked on as well.

Reply:
Regression fixed